db/migrate/20160218161746_add_csv_export.rb

```diff
@@ -0,0 +1,12 @@
+class AddCsvExport < ActiveRecord::Migration
+  def change
+    create_table :csv_exports do |t|
+      t.integer :user_id,    null: false
+      t.timestamps           null: false
+      t.timestamp :deleted_at
+      t.string :content,        null: false
```

Comment:
To identify the type of export.  Was leaving room to move users csv export if that ever becomes thousands of lines and too slow to generate real-time.  With the move to project level authorizations and more deployers, could reach a scenario where the report generates several thousand row entries that may require asynchronous exports.